src/Utils/__test__/errors.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as Sentry from "@sentry/browser"
+import { isErrorInfo, sendErrorToService } from "Utils/errors"
+
+jest.mock("@sentry/browser")
+
+describe("errors", () => {
+  describe("#isErrorInfo", () => {
+    it("returns true if an object implements the ErrorInfo interface", () => {
+      const errorInfo = { componentStack: "someString", anotherKey: 0 }
+      expect(isErrorInfo(errorInfo)).toBe(true)
+
+      const notErrorInfo = { anotherKey: 0 }
+      expect(isErrorInfo(notErrorInfo)).toBe(false)
+    })
```

Comment:
This seems like it's testing two things, can we break it up into two `it()` blocks?

Reply:
Done!